frontend/components/ProfilePublication/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+
+import Publication from './Publication'
+
+const datas = [
+  { title: 'Publication1', desc: 'Description1!', link: 'google.com' },
+
+  { title: 'Publication2', desc: 'Description2!', link: 'google.com' }
+]
+
+class ProfilePublication extends Component {
+  render() {
+    return (
+      <>
+        {style}
+        <div className="grey lighten-1 profileContent">
+          <div className="col s12" style={{ padding: 0 }}>
+            <h4 style={{ margin: '0', color: '#0f6489' }}>Publications</h4>
+            <div>
+              {datas.map(data => (
+                <Publication data={data} />
+              ))}
```

Comment:
Great use of loops for repetitive structure !!

Reply:
You need to specify a `key` every time you use this kind of loop to help react keep tract of all the elements